tests/serialization/test_parquet.py

```diff
@@ -35,46 +37,6 @@ def test_timestamp_us(store):
     pdt.assert_frame_equal(DataFrameSerializer.restore_dataframe(store, key), df)
 
 
-def test_pyarrow_07992(store):
-    key = "test.parquet"
-    df = pd.DataFrame({"a": [1]})
-    table = pa.Table.from_pandas(df)
-    meta = b"""{
-        "pandas_version": "0.20.3",
-        "index_columns": ["__index_level_0__"],
-        "columns": [
-            {"metadata": null, "name": "a", "numpy_type": "int64", "pandas_type": "int64"},
-            {"metadata": null, "name": null, "numpy_type": "int64", "pandas_type": "int64"}
-        ],
-        "column_indexes": [
-            {"metadata": null, "name": null, "numpy_type": "object", "pandas_type": "string"}
-        ]
-    }"""
-    table = table.replace_schema_metadata({b"pandas": meta})
-    buf = pa.BufferOutputStream()
-    pq.write_table(table, buf)
-    store.put(key, buf.getvalue().to_pybytes())
-    pdt.assert_frame_equal(DataFrameSerializer.restore_dataframe(store, key), df)
-
-
-def test_index_metadata(store):
```

Comment:
This test was also related to the arrow 0.7.992 version (this version was only used BY internally)

Reply:
What's wrong with keeping it?